apps/docs/content/components/Button/index.mdx

```diff
@@ -73,6 +73,12 @@ Subtle buttons have a transparent background with translucent hover and active s
 
 ![An image showing all subtle states.](https://github.com/primer/brand/assets/912236/7420969a-e561-4fd6-9bef-7340e2b0a7e8)
 
+#### Block
+
+Turning on the block option for button allows the button to span the full width of its container. This feature can be particularly useful when you want the button to occupy more space. Block button can continue to pair with secondary or primary variants.
+
+![An image showing block button with all subtle states.](https://github.com/primer/brand/assets/912236/7420969a-e561-4fd6-9bef-7340e2b0a7e8)
```

Comment:
👋 @jesskuo4 - just a reminder about this image. It's not showing the same image as subtle example above it. Could you swap out for the block version please?

![Screenshot 2023-08-08 at 16 21 15](https://github.com/primer/brand/assets/13340707/1174b50e-2a24-47c6-86de-c74360ea0d38)

Reply:
sounds good! thank you @rezrah